src/vm.rs

```diff
@@ -817,6 +825,65 @@ mod tests {
 		assert!(freq > 0);
 		assert!(freq < 10000); //More than 10Ghz is probably wrong
 	}
+
+	#[cfg(target_os = "linux")]
```

Comment:
Is there any specific reason the test is valid for linux?

Reply:
macOS is currently not fully supported. In addition, it is only possible to create one VM with the macOS hypervisor extension. Your test do at least twice.